src/BaseTemplates/StarterWeb/wwwroot/lib/jquery-validation-unobtrusive/.bower.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I am not clear why this file is checked in...this should be auto generated when bower installs the packages?